config/runtime/server.go

```diff
@@ -609,53 +609,44 @@ func configureAccessControls(conf *config.Couper, confCtx *hcl.EvalContext, log
 		for _, jwtConf := range conf.Definitions.JWT {
 			confErr := errors.Configuration.Label(jwtConf.Name)
 
-			var jwt *ac.JWT
+			jwtOptions := &ac.JWTOptions{
```

Comment:
Maybe we can improve this further with extracting this block to a newJWTOpts method.

Reply:
As the JWT access control is created with different functions depending on the JWT configuration (as is the JWTOptions), I moved the creation of the _JWT AC_ (not the JWTOptions) to a function. OK?